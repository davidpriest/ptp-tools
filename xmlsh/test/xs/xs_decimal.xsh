#
# Test of xs:integer
# Test of xs:integer

import commands xs=xs

d=xs:decimal( 25.2 )
xtype $d
echo $d

d=xs:decimal( <[ xs:decimal( 2.54 ) ]> )
xtype $d
echo $d

d=xs:decimal( xs:string(.001) )
xtype $d
echo $d


d=xs:decimal( xs:integer(1) )
xtype $d
echo $d




