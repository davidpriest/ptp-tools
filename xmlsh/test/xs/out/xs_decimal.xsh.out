xs:decimal
25.2
xs:decimal
2.54
xs:decimal
0.001
xs:decimal
1
